### PR TITLE
Add DSTU 2 documentation for Observation.

### PR DESCRIPTION
**Pull request by jfrhong**

### CODE REVIEW

**Comment by cerbs** [comment]
:+1:

File: content/dstu2/observation.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Search for labs, vitals, and alcohol/tobacco use Observations that meet supplied query parameters:
+
+    GET /Observation?:parameters
+
+### Parameters
+
+ Name            | Required?| Type                                                           | Description
+-----------------|----------|----------------------------------------------------------------|-------------------------------------------------------
+`patient`        | Either this or `subject` |[`reference`](http://hl7.org/fhir/dstu2/search.html#reference)| The subject that the observation is about (if patient). EG: `123456`
```
Comment (MaxPhilips) [inline content/dstu2/observation.md]: In @davischristi 's DiagnosticReport PR he wrote these as "This or subject".  We should use a consistent pattern, I have no real preference on which one

https://github.com/cerner/fhir.cerner.com/pull/42/files#diff-a1330e0905fd78f90fc132493e2caf4eR20

**Comment by MaxPhilips** [comment]
+1

File: content/dstu2/observation.md (hunk 1 above)
Comment (jfrhong) [inline content/dstu2/observation.md]: Removed 'either'

**Pull request merged**